opm/input/eclipse/Schedule/UDQ/UDQActive.hpp

```diff
@@ -185,16 +411,22 @@ class UDQActive {
     }
 
 private:
-    std::string udq_hash(const std::string& udq, UDAControl control);
-    std::string wg_hash(const std::string& wgname, UDAControl control);
-    int add(const UDQConfig& udq_config, const std::string& udq, const std::string& wgname, UDAControl control);
-    int update_input(const UDQConfig& udq_config, const UDAValue& uda, const std::string& wgname, UDAControl control);
-    int drop(const std::string& wgname, UDAControl control);
-
-    std::vector<InputRecord> input_data;
-    std::vector<OutputRecord> mutable output_data;
+    /// Current set of UDAs entered in the input source.
+    std::vector<InputRecord> input_data{};
+
+    /// Current set of UDAs condensed by use counts and IUAP start pointers.
+    ///
+    /// Intended for restart file output as the IUAD vector.  Cleared if
+    /// input_data changes.  Formed in constructOutputRecords().
+    ///
+    /// Marked 'mutable' becase we may need to construct this from the
```

Comment:
because

Reply:
Thanks.  I've pushed an update to address this.